src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
+  'hasCapability', false, 'packageAPI'
+);
+
+if (shouldRenderUniverseOption) {
```

Comment:
@MatApple keep in mind that we already use the `hasCapability` filter in other areas. We should abstract the call of the filter event because it's odd to have these littered all over the place. This is out of scope though.